renovate.json

```diff
@@ -74,6 +74,18 @@
       ],
       "matchUpdateTypes": ["major"],
       "enabled": false
+    },
+    {
+      "description": "Restrict AutoMapper due to incompatibility with IdentityServer4.EntityFramework.Storage",
```

Comment:
Part of me thinks it might be easier to just get rid of AutoMapper completely. This is not to say we shouldn't upgrade Identity Server though.

Personally, I'd be all for just hand-rolling our mappings, but if not, there are obviously other libraries we could use (e.g. Mapperly maybe).

Reply:
I agree, I'm up for stopping using it. I would already replace simple mappings right now and probably just leave the complicated ones where we save tons of lines by using a mapper, until we reach a decision going forward.

Without upgrading IdentityServer we are also exposed to old transitive dependencies (like we'd still be including an old version of AutoMapper indirectly).